src/frontend/src/pages/CalendarPage/DesignReviewDetailPage/FinalizeDesignReviewModal.tsx

```diff
@@ -0,0 +1,56 @@
+import NERModal from '../../../components/NERModal';
+
+interface FinalizeDesignReviewProps {
+  open: boolean;
+  setOpen: (val: boolean) => void;
+  onSubmit?: () => void;
+}
+
+const FinalizeDesignReviewModal = ({ open, setOpen, onSubmit }: FinalizeDesignReviewProps) => {
+  const handleClose = () => {
+    setOpen(false); // Close the modal by setting open to false
+  };
+
+  return (
+    <NERModal
+      open={open}
+      title={''}
+      onHide={handleClose} // Call handleClose function to close the modal
+      onSubmit={onSubmit}
+    >
+      <div>
```

Comment:
We don't want to use divs or normal HTML tags, try using grids and custom components instead